src/Microsoft.Framework.CommandLineUtils/CommandLine/AnsiConsole.cs

```diff
@@ -15,6 +15,8 @@ private AnsiConsole(TextWriter writer)
             OriginalForegroundColor = Console.ForegroundColor;
         }
 
+        private int BoldRecursion = 0;
```

Comment:
Fields --> _boldRecursion.

Also, no need to initialize to 0 because that's the default value of an int.